src/game/default/bg_pmove.c

```diff
@@ -1176,8 +1180,14 @@ static void Pm_SpectatorMove() {
  */
 static void Pm_Init(void) {
 
-	VectorScale(PM_MINS, PM_SCALE, pm->mins);
-	VectorScale(PM_MAXS, PM_SCALE, pm->maxs);
+	// set the default bounding box
+	if (pm->s.flags & PMF_GIBLET) {
+		VectorCopy(PM_GIBLET_MINS, pm->mins);
+		VectorCopy(PM_GIBLET_MAXS, pm->maxs);
+	} else {
+		VectorScale(PM_MINS, PM_SCALE, pm->mins);
+		VectorScale(PM_MAXS, PM_SCALE, pm->maxs);
+	}
```

Comment:
PMF_GIBLET constitutes its own bounding box. I suppose PM_DEAD could be incorporated here, actually.